docs/en/setup/backend/backend-zabbix.md

```diff
@@ -0,0 +1,70 @@
+# Zabbix Receiver
```

Comment:
Do we have some benchmark about this? And which authentication and security are we supporting now?

Reply:
It using TCP short connection. no plan to add a benchmark. And no authentication support currently.